ee/clickhouse/models/cohort.py

```diff
@@ -17,14 +17,14 @@
 from posthog.models import Action, Cohort, Filter, Team
 
 
-def format_person_query(cohort: Cohort) -> Tuple[str, Dict[str, Any]]:
+def format_person_query(cohort: Cohort, custom_match_field="person_id") -> Tuple[str, Dict[str, Any]]:
```

Comment:
Do we want to allow any string as a custom match field? If not, let's create a union type for it above.

Reply:
I just changed the param to a kwarg. Should it still be a union type?